README.md

```diff
@@ -11,11 +11,27 @@ one.
 > You might encounter issues in production, since this project is currently in
 > **alpha**.
 
-Quick Links:
-
-* [How does it work](#how-does-it-work)
-* [Set it up on Cloud Run](#setup)
-* [Try it out (locally)](#try-out)
+## Table of Contents
+
+<!-- toc -->
```

Comment:
might be worth  adding a note like this:
https://github.com/ahmetb/cloud-run-faq/blob/c635724fc8f74d07468d5c40db5744afc3fb3afa/README.md#L32-L39